src/components/Wikipage.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Logo from "./Logo";
+import Searchbar from "./Searchbar";
+import { searchWiki } from "../api";
+
+interface Props {
+    onSearch: (text: string) => void;
+}
```

Comment:
I see you have used the props interface in the component itself. Well there's no harm in it (I sometimes do that too 😬 ) but a good practice is to put it in a separate file under the component folder. So it could've been like


./Wikipage
  - Wikipage.tsx (or index.tsx)
  - Wikipage.props.ts

Reply:
also, a usual practive is to put `I` prefix behind the interfaces. So I would've named it `IWikipageProps`